src/components/AccountIcon/index.jsx

```diff
@@ -23,6 +24,9 @@ const _AccountIcon = ({ account, className, size }) => {
   if (!institutionSlug) {
     return null
   }
+  if (flag('banks.perf.deactivate-account-icons')) {
```

Comment:
Can you explain why we need these flags?

Reply:
Yep, this are development flags to check performance to check if perfomance improves if we show/hide account icons. I suspect account icons to have an impact since they trigger many requests and are not properly cached.

I can remove them since they are not necessary for this particular PR.

Reply:
I've removed those flags.

Reply:
Thx for the explanation